lib/components/ExpandableCategoryList.dart

```diff
@@ -1,53 +1,59 @@
 import 'package:flutter/material.dart';
 import "package:collection/collection.dart";
 
-class ExpandableCategoryList<TCategory, TItem> extends StatelessWidget {
+class ExpandableCategoryList<TCategory extends Object, TItem extends Object>
+    extends StatelessWidget {
```

Comment:
It was an amazing work and I'm proud of you, but you replace the category list, and it is not used anymore, you can remove this entire file